projects/kit/components/input-date-time/input-date-time.component.ts

```diff
@@ -321,4 +312,15 @@ export class TuiInputDateTimeComponent
             guide: false,
         };
     }
+
+    @tuiPure
+    private getDateTimeString(
+        date: TuiDay,
+        time: TuiTime | string | null,
+        {timeMode = 'HH:MM'}: {timeMode?: TuiTimeMode} = {},
```

Comment:
Why an object?

Reply:
third argument is all possible `options` for `getDateTimeString` (with only one `timeMode ` now ).

I can remove object if you think that it is overengineering

Reply:
Yes, let's keep it simple for now, especially since it's a private method.

Reply:
i removed it